app/models/opponent.rb

```diff
@@ -36,6 +36,26 @@ class Opponent < ApplicationRecord
     interested_party: 'Interested party'
   }.freeze
 
+  def self.dummy_opponent
+    Opponent.find_by(other_party_id: 'OPPONENT_00000001') || create_dummy_opponent
+  end
+
+  def self.create_dummy_opponent
+    Opponent.create(
+      other_party_id: 'OPPONENT_00000001',
```

Comment:
Unless you decide to use `find_or_create_by!`, I think this string should be a constant as it is repeated twice.
Does it have to be `OPPONENT_00000001` ? Would `OPPONENT_1` be a bit simpler?

Reply:
It does have to be OPPONENT_nnnnnnnn I'm afraid.  In fact, this is a value that is going to be dictated by CCMS - a different value for each environment, so there will be more work to do in this area, but this will suffice for now.